src/envoy/http/authn/authenticator_base_test.cc

```diff
@@ -156,6 +156,7 @@ TEST_F(AuthenticatorBaseTest, ValidateJwtWithJwtInHeader) {
                "audiences": ["bookstore-esp-echo.cloudendpointsapis.com"],
                "presenter": "",
                "claims": {
+                 "aud": "bookstore-esp-echo.cloudendpointsapis.com",
                  "iss": "628645741881-noabiu23f5a8m8ovd8ucv698lj78vv0l@developer.gserviceaccount.com",
                  "sub": "628645741881-noabiu23f5a8m8ovd8ucv698lj78vv0l@developer.gserviceaccount.com"
```

Comment:
We can avoid using these long claim values by generating a JWT payload for test. 
There are examples in proxy/src/envoy/http/jwt_auth/jwt_test.cc, and scripts in proxy/src/envoy/http/jwt_auth/tools/.
I am okay to leave it as is for now, but if we are going to add more tests, then it is better to keep these claims short for good readability.

Reply:
Done: long claim values are shortened.